frontend/components/initiative/Toolbar.vue

```diff
@@ -0,0 +1,76 @@
+<script setup lang="ts">
+import { type InitiativeUserRelationship } from '@/openapi/generated/pacta'
+
+const localePath = useLocalePath()
+const { getMaybeMe } = useSession()
+const { t } = useI18n()
+
+const { isAdmin, maybeMe } = await getMaybeMe()
+const prefix = 'InitiativeToolbar'
+const tt = (key: string) => t(`${prefix}.${key}`)
+
+interface Props {
+  initiativeId: string
+  initiativeUserRelationships: InitiativeUserRelationship[]
+}
+const props = defineProps<Props>()
+
+const isManager = computed(() => {
+  const mm = maybeMe.value
+  return (!!mm && props.initiativeUserRelationships.some(r => r.manager && r.userId === mm.id)) || true
+})
+const isMember = computed(() => {
+  const mm = maybeMe.value
+  return !!mm && props.initiativeUserRelationships.some(r => r.member && r.userId === mm.id)
+})
+const canEdit = computed<boolean>(() => isManager.value || isAdmin.value)
+const canSeeInternal = computed<boolean>(() => isManager.value || isMember.value || isAdmin.value)
+const showToolbar = computed<boolean>(() => canSeeInternal.value || canEdit.value)
```

Comment:
As it currently stands, `showToolbar` is logically equivalent to `canSeeInternal`, but feel free to leave if you expect them to diverge.

Reply:
I've found with AS it's been easier for us to have sentence-type computeds for frontend visibility gates. It's one extra line of code, but decouples the two from one another by default (i.e. modifying the visibility for one wouldn't require a change to the code for the other).